mod-orders-storage/schemas/receiving_history.json

```diff
@@ -46,6 +51,11 @@
       "type": "string",
       "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
     },
+    "receiptStatus": {
```

Comment:
There is `receivingStatus` property is already defined which represents `piece` record status. The `receiptStatus` is quite similar but actually describes PO Line's receipt status. I think better to use `poLine` prefix for the PO Line's properties.